setup.sh

```diff
@@ -0,0 +1,5 @@
+git add .
```

Comment:
Oh that one - Basically I made it as a convenience tool to deploy on Heroku. Basically instead of doing those 4 things one by one, I simply bash it on heroku.
The script is not actually on heroku itself as I have removed it from the staging area. If I need to think before committing I will not bash this script